plenum/server/domain_req_handler.py

```diff
@@ -163,32 +164,44 @@ def getNymDetails(state, nym, isCommitted: bool=True):
     def nym_to_state_key(nym: str) -> bytes:
         return sha256(nym.encode()).digest()
 
-    def make_proof(self, path, head_hash=None):
+    def get_value_from_state(self, path, head_hash=None, with_proof=False):
```

Comment:
Do we have the corresponding PR in Indy-node?

Reply:
Not a PR but branch

Reply:
Can you please raise a PR since the PR in the node needs to be merged right after the one in plenum?

Reply:
Here, https://github.com/hyperledger/indy-node/pull/730